src/validation.js

```diff
@@ -123,17 +123,19 @@ function validateColumnName(name) {
  * Validates a designated timestamp. The value must contain only digits.<br>
  * Throws an error if the value is invalid.
  *
- * @param {string} timestamp - The table name to validate.
+ * @param {string | bigint} timestamp - The table name to validate.
```

Comment:
nit: not something introduced in this PR, but would be nice to fix the doc.
"The table name to validate" -> "The timestamp value to validate"

Reply:
Good spot. Fixed in d7e67f5ed0cecda5e1ff5a4a2ed1a347c4b4875b